src/ontology/mondo.Makefile

```diff
@@ -1299,6 +1303,44 @@ update-subclass-sync: $(TMPDIR)/subclass-confirmed.robot.owl tmp/subclass-axioms
 		mv NORM $(SRC)
 
 
+# All the synchronized sources
+SYNCED_SOURCES := DOID ICD10CM ICD10WHO icd11.foundation NCIT OMIM OMIMPS Orphanet
+
+# Join the sources with '|' to form a regex for use in commands
+SOURCES_REGEX := $(shell IFS='|'; echo "$(SYNCED_SOURCES)" | sed 's/ /|/g')
+
+# This target extracts all synonyms from mondo edit and then drops all axiom
+# annotations related to the sources that are in the list of curated sources
+tmp/synonyms-axioms.owl: $(SRC)
+	$(ROBOT) filter --input $(SRC) \
+		--term oboInOwl:hasExactSynonym \
+		--term oboInOwl:hasNarrowSynonym \
+		--term oboInOwl:hasBroadSynonym \
+		--term oboInOwl:hasRelatedSynonym \
+		--axioms annotation --preserve-structure false --trim false \
+		--drop-axiom-annotations "oboInOwl:source=~'($(SOURCES_REGEX)):.*'" \
+		-o $@
+
+# This command updates mondo-edit with all the confirmed synonyms evidence from the mondo-ingest repo
+.PHONY: update-synonyms-sync
+update-synonyms-sync: $(TMPDIR)/synonyms-confirmed.robot.owl tmp/synonyms-axioms.owl
+	$(ROBOT) \
+		remove --input $(SRC) \
```

Comment:
## Dropping?
@matentzn Can you explain this part?

It looks like it is dropping all existing synonym annotations before re-adding the confirmed ones and their source annotations?

If it is really dropping all of them, aren't there situations in which we have a synonym or different scope that is the result of manual curation and we don't want to destroy that?

Reply:
You need to check this target: tmp/synonyms-axioms.owl; it contains all (!) synonyms in mondo-edit, with their provenance removed. That is also being merged back in! So this process will never drop anything!

Reply:
#### 1. What do you mean by _(!)_?

#### 2. Unconfirmed source not being dropped?
I did look at that target, but is it doing what it is supposed to?

Observe this case. Checking the diff in #8270, `MONDO:0000004`, this line was unmodified:
`synonym: "adrenal cortical insufficiency" EXACT [DOID:10493, NCIT:C26691, NCIT:C27150]`

But in the [confirmed cases](https://docs.google.com/spreadsheets/d/1ftEZ47-JDe-1Z_fRWyO5NS39qT_-8MZsHXHEKz9aZ9Q/edit?gid=978570179#gid=978570179), `NCIT:C27150` is not a confirmed case:

|mondo_id     |mondo_label                 |synonym_scope_source|synonym_scope_mondo|synonym                       |source_id  |
|-------------|----------------------------|--------------------|-------------------|------------------------------|-----------|
|MONDO:0000004|adrenocortical insufficiency|oio:hasExactSynonym |                   |adrenal cortical insufficiency|DOID:10493 |
|MONDO:0000004|adrenocortical insufficiency|oio:hasExactSynonym |                   |adrenal cortical insufficiency|NCIT:C26691|

So shouldn't provenance for `NCIT:C27150` be removed?

Reply:
Oooppppsss. see https://github.com/monarch-initiative/mondo/commit/b87630ec7d56d82aeaf28e2f06e6715d393d63d5

Copy paste error :D THANKS for seeing that, thank Open Science for reviews!!

Reply:
(Updated the build as well)

Reply:
Thanks! Resolving.